h/search/client.py

```diff
@@ -42,28 +38,35 @@ def conn(self):
         return self._conn
 
 
-def get_client(settings):
-    """Return a client for the Elasticsearch index."""
-    host = settings['es.host']
-    index = settings['es.index']
-    kwargs = {}
-    kwargs['max_retries'] = settings.get('es.client.max_retries', 3)
-    kwargs['retry_on_timeout'] = settings.get('es.client.retry_on_timeout', False)
-    kwargs['timeout'] = settings.get('es.client.timeout', 10)
+def _get_settings(settings, elasticsearch=elasticsearch):
+    kwargs = {'verify_certs': True,
+              'max_retries': settings.get('es.client.max_retries', 3),
+              'retry_on_timeout': settings.get('es.client.retry_on_timeout', False),
+              'timeout': settings.get('es.client.timeout', 10)}
```

Comment:
Rather than refactoring the existing code to share things between the old and new code, I'd rather just duplicate these settings in two places. We're going to be deleting the old code anyway.